tests/Webapi/Dom/Webapi__Dom__DataTransferItem__test.res

```diff
@@ -0,0 +1,9 @@
+open Webapi.Dom
+open DataTransferItem
+
+let item = DataTransfer.make()->DataTransfer.items->DataTransferItemList.get(0)->Js.Option.getExn
+
+let kind = item->kind
```

Comment:
I changed the style of these getters from `let _ = item->kind` to `let kind = item->kind` this way they are not compiled away and show up in the JS output that we also commit easier to see then what the bindings generate.

Reply:
yes, that's #42